src/test/rgw/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ target_include_directories(ceph_test_rgw_redis_lock
   ${rgw_libs}
   ${UNITTEST_LIBS}
   ${EXTRALIBS}
+  Boost::json
```

Comment:
Thankyou @yuvalif ! 
Should I use boost JSON? 

I will have to cherry-pick your commit as I do not want my last commit and have a commit ready to overwrite that.

Reply:
you can use boost json (as it already a dependency due to boost redis) or pico json.
whatever is easier to use